src/pages/journal/journal.reducer.ts

```diff
@@ -21,6 +22,13 @@ export function journalReducer(state = initialState, action: journalActions.Jour
         isLoading: true,
         error: { message: '', status: 0, statusText: '' },
       };
+    case journalActions.CANDIDATE_DETAILS_SEEN:
+      return {
+        ...state,
+        checkComplete: state.checkComplete.concat({
```

Comment:
Minor stylistic thing, but we tend to use spread instead of `concat` in reducers.